docker/Makefile

```diff
@@ -20,9 +20,16 @@ DOCKER_GROUP_ID=$(shell stat --format "%g" $(shell readlink -f /var/run/docker.s
 NE_GID := $(shell getent group ne | awk -F: '{printf $$3}')
 endif
 
+ifeq ($(ARCH),x86_64)
+DOCKER_ARCH = amd64/
```

Comment:
The trailing `/` is a bit unexpected. Is it necessary?

Reply:
Yes. The Dockerfile does not support the case where the `/` is not in the variable.